Gemfile

```diff
@@ -0,0 +1,48 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.7.0"
+
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+
+gem "sqlite3", "~> 1.4"
```

Comment:
remove this gem
add gem "mysql2"
after that, you bundle install and restart server

Reply:
remove both gem?